istio-start.adoc

```diff
@@ -0,0 +1,67 @@
+// =================================================================================================
+// Deploying Istio
+// =================================================================================================
+
+== Deploying Istio
+
+Install {istio} by following the instructions in the official https://istio.io/latest/docs/setup/getting-started[{istio} Getting started^] documentation.
+
+Run the following command to verify that `istioctl` path has been set successfully:
+
+[role=command]
+```
+istioctl version
+```
+
+The output will be similar to the following example.
+[source, role="no_copy"]
+----
+no running Istio pods in "istio-system"
+1.7.3
+----
+
+We will use the following command to configure the {istio} profile on {kube}
```

Comment:
@tt-le Should not use the first person words `We`, `I`. `our`, etc. Suggest "We will use..." -> "Use ..."